apps/trading-e2e/src/support/trading-windows/trades-list.ts

```diff
@@ -0,0 +1,23 @@
+export default class TradesList {
+  colIdPrice = 'price';
+  colIdSize = 'size';
+  colIdCreatedAt = 'createdAt';
+
+  verifyTradesListDisplayed() {
+    cy.get(`[col-id=${this.colIdPrice}]`).each(($tradePrice) => {
+      cy.wrap($tradePrice).invoke('text').should('not.be.empty');
+    });
+    cy.get(`[col-id=${this.colIdSize}]`).each(($tradeSize) => {
+      cy.wrap($tradeSize).invoke('text').should('not.be.empty');
+    });
+
+    const dateTimeRegex =
```

Comment:
nitpick (non blocking): Could just import in our date formatter function here

Reply:
I could be wrong but I think using regex to assert here is probably a bit more efficient.